content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -100,7 +46,7 @@ follow
 
 ## Objectives
 
-* Install a secure Kubernetes cluster on your machines
+* Install a best practices Kubernetes cluster on your machines
```

Comment:
Might be better to say: Install a Kubernetes cluster using recommended best practices  on your machines.